### PR TITLE
Added Youtube Link field to Challenge and displaying it in the timathon history page

### PR DESCRIPTION
**Pull request by KrishnaKanth1729**
run `python manage.py makemigrations` and `python manage.py migrate`
and add all Youtube Video links of Tim's videos in database

### CODE REVIEW

**Comment by KrishnaKanth1729** [comment]
it is for displaying the link in the timathon history page :cheemsadidas:

File: TWT/apps/challenges/models/challenge.py
Comment (takos22) [inline TWT/apps/challenges/models/challenge.py:62]: Why `max_length=43`?

File: TWT/templates/timathon/codejam_listView.html
Comment (takos22) [inline TWT/templates/timathon/codejam_listView.html:22]: This would be better if it were a button

File: TWT/apps/challenges/models/challenge.py
Comment (KrishnaKanth1729) [reply TWT/apps/challenges/models/challenge.py:62]: because that's the length of youtube links

File: TWT/templates/timathon/codejam_listView.html
Comment (KrishnaKanth1729) [reply TWT/templates/timathon/codejam_listView.html:22]: ok so shall i add a button?

**Review by Nisarg0007 — APPROVED** [review]

**Review by takos22 — APPROVED** [review]
LGTM, waiting for other core dev reviews before merging

File: TWT/apps/challenges/models/challenge.py
Comment (Shubhaankar-Sharma) [reply TWT/apps/challenges/models/challenge.py:62]: why not models.URLField?

File: TWT/templates/timathon/codejam_listView.html
Comment (Shubhaankar-Sharma) [reply TWT/templates/timathon/codejam_listView.html]: ur choice. but add custom css to make it look like a button

File: TWT/apps/challenges/models/challenge.py
Comment (Shubhaankar-Sharma) [reply TWT/apps/challenges/models/challenge.py:62]: wait actually nvm this is fine

**Comment by Shubhaankar-Sharma** [comment]
No migrations? run `python manage.py makemigrations`

**Review by Shubhaankar-Sharma — CHANGES_REQUESTED** [review]
migrations & testing if this works? I know this works, but still before being deployed it needs testing, if you arent able to set this up let me know, ill do makemigrations and

File: TWT/templates/timathon/codejam_listView.html
Comment (KrishnaKanth1729) [reply TWT/templates/timathon/codejam_listView.html]: i have made it as a button. see the next commit. i have used to info bootstrap button :mhm: @Shubhaankar-Sharma

**Comment by KrishnaKanth1729** [comment]
@Shubhaankar-Sharma I have problems setting it up. So if u can do migrations that'd be good. :smilecat:

**Pull request merged**
